index.bs

```diff
@@ -519,10 +525,12 @@ Individuals Representing a Member Organization</h5>
 	the number of individuals representing a W3C Member
 	(or group of <a href="#MemberRelated">related Members</a>).
 
-<h4 id="GeneralMeetings">
+<h4 id="GeneralMeetings" oldids="GAEvents, EventsS">
 Meetings</h4>
 
-	The requirements in this section apply to the official meetings of any [=W3C group=].
+	The requirements in this section apply to the official meetings of any [=W3C group=],
+	as well as to official W3C meetings with open-ended participation from the Membership and/or the public,
+	such as [=Workshops=] or other events open to the whole [=Member|Membership=] or to the public.
```

Comment:
The text here is somewhat redundant.

Reply:
Indeed. That was stray text left after a rewrite. Fixed.